about/releases/changelogs/0.20.1/index.md

```diff
@@ -0,0 +1,74 @@
+# 0.20.1
+  - Release Date: 2024-07-18
+  - Commit: [b368116](https://github.com/OpenSpace/OpenSpace/commit/c26f9db7d427c9d72f339f745cb2e5fb243ca6aa)
+  - Full changelog: [link](https://github.com/OpenSpace/OpenSpace/releases/tag/releases%2Fv0.20.1)
+
+Download version 0.20.1 for Windows and Mac on the OpenSpace website [installation page](https://openspaceproject.com/version-0201). Below are notes that highlight new content and bug fixes that will be relevant for OpenSpace users.
```

Comment:
```suggestion
Download version 0.20.1 on the OpenSpace website [installation page](https://openspaceproject.com/version-0201). Below are notes that highlight new content and bug fixes that will be relevant for OpenSpace users.
```

Since we don't even have a mac build for 0.20.0, we shouldn't mention it here, i think

Reply:
I agree.  Didn't know that we didn't make one for 0.20.0

Although, this taxt is copied from the page for 0.20.0, so I think we should remove it there too then